koku/api/migrations/0017_auto_20180808_2134.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1 on 2018-08-08 21:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0016_auto_20180802_1911'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='provider',
+            name='type',
+            field=models.CharField(choices=[('AWS', 'AWS')], default='AWS', max_length=50),
```

Comment:
Why did we eliminate the local provider type?

Reply:
Sorry, I missed this change. I didn't focus very hard on the migrations, but we definitely don't want the local provider removed.